slices/git.yaml

```diff
@@ -0,0 +1,41 @@
+package: git
+
+essential:
+  - git_copyright
+
+slices:
+  bins:
+    essential:
+      - libc6_libs
+      - libpcre2-8-0_libs
+      - zlib1g_libs
+    contents:
+      /usr/bin/git:
+      /usr/bin/git-shell:
+      /usr/bin/scalar:
+
+  core:
+    essential:
+      - libc6_libs
+      - libcurl3t64-gnutls_libs
+      - libexpat1_libs
+      - libpcre2-8-0_libs
+      - zlib1g_libs
+      - git_bins
+    contents:
+      /usr/lib/git-core/**:
+
+  contrib:
+    essential:
+      - git_perl5
+    contents:
+      /usr/share/doc/git/contrib/**:
```

Comment:
These two paths would probably break since the symlink target might not exist:
```
lrwxrwxrwx root/root         0 2024-05-20 11:15 ./usr/share/doc/git/contrib/persistent-https/LICENSE -> ../../../../common-licenses/Apache-2.0
lrwxrwxrwx root/root         0 2024-05-20 11:15 ./usr/share/doc/git/contrib/subtree/COPYING -> ../../../../common-licenses/GPL-2
```